requirements.txt

```diff
@@ -2,4 +2,4 @@ click
 rlp>=1.0.0
 coincurve>=7.1.0,<8.0
 web3>=4.0.0
-py-solc
\ No newline at end of file
+py-solc==2.1.0
```

Comment:
Title says 2.2.0, but change is 2.1.0. Which one is the intended correct version?

Reply:
`2.1.0` actually, `2.2.0` also doesn't work. I'll rename the PR.